package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "wizard-sdk",
+  "version": "0.0.1",
+  "description": "typescript sdk to visualize EIP-712 various protocols, simulate and decode transactions",
+  "main": "index.js",
+  "repository": "git@github.com:trustwallet/wizard-sdk.git",
+  "author": "Abdou <118254027+a6-dou@users.noreply.github.com>",
+  "license": "MIT",
```

Comment:
Also BSD, it's similar to MIT with trademark protection

Reply:
should we update the `LICENSE` file at `root` dir also? currently it's `MIT`